tests/buyers/views/test_buyers.py

```diff
@@ -1969,6 +1969,14 @@ def setup_method(self, method):
                 "blah": [False, True]
             }
         ]
+        self.bad_response = {
+            "supplierName": "Tea Boy Ltd.",
+            "respondToEmailAddress": "teaboy@example.com",
+            "availability": "Tomorrow",
+            "essentialRequirementsMet": False,
```

Comment:
After some further digging, because of this the user could never 'submit' a brief response that had essential requirements met as False. Therefore because the brief response is not submitted it will not be returned when you request all brief responses from the API (unless you specified otherwise).